header-footer-grid/Core/Components/Logo.php

```diff
@@ -102,6 +264,39 @@ public function add_settings() {
 			);
 		}
 
```

Comment:
@preda-bogdan Not a must if you feel that there's too much to change or technical limitations that wouldn't make it worth it. 

What do you think about using multiple settings for the toggle & dark logo instead of using a JSON? This way we make sure there's nothing to migrate and this won't yield weird results.

We do that for the font variants selector. Essentially you'd have multiple values managed by the same UI control:

https://github.com/Codeinwp/neve/blob/master/inc/customizer/options/typography.php#L109-L116
https://github.com/Codeinwp/neve/blob/master/assets/apps/customizer-controls/src/font-family/FontFamilyComponent.js#L104-L107